### DISCUSSION TITLE
Bitlocker template not supported?

### DISCUSSION THREAD

**Discussion by yannara** [Micke-K/IntuneManagement#221] [Ideas]
I noticed, that new Bitlocker configuration (MS did some changes like 3 month ago to Disk Encryption configuration), is not viewed to be exported. Other old Endpoint Security and Settings Catalog I see, but not Bitlocker.

**Comment by Jien-hub** [comment]
Same here for the new Security Baselines. The one for 22H2 show but not for 22H3

**Comment by Micke-K** [comment]
Hello!

New Endpoint Security policies should show up in Settings Catalog. I can see the 23H2 policies there. I haven't checked BitLocker but I'll have a look when I get the chance. Microsoft is moving away from the Intent API used by Endpoint Security policies to the Settings Catalog API. 

Cheers!

**Reply by Jien-hub** [reply]
Got it! Can confirm it shows up in Settings Catolog. Love this tool!

**Reply by yannara** [reply]
I too see it now, love this tool more :D

**Comment by Micke-K** [comment]
Hello,
I noticed that Settings Catalog will mot return all items for some reason. I have ~20 policies and only ~10 or so are returned. This should return 100 so not sure what's happening on the server side. 

If not all items are returned, there should be buttons visible to load more or load all.

Cheers!

**Reply by yannara** [reply]
Load all works fine, no problem. I have more than 20 setting catalog objects.

**Reply by joshuahipple** [reply]
Thanks, I confirmed the "Load All" worked for me as well!!  Reviewing the new baselines now.
